GameBox/GameBox/ECS/Systems/AISystem.h

```diff
@@ -23,9 +25,17 @@ struct AIComponent
 	// Ball entity
 	AI::State state = AI::State::IDLE;
 	sf::Vector2f targetPos;
+	entityx::Entity targetEntity;
 	float speed = 100.0f;
-	float attackspeed = 3.0f;
+	float attackspeed = 1.0f;
+	float coolDownTimer = 0.0f; // used in state machine
 	std::vector<sf::Vector2f> targetPosVector;
+	uint32_t attackRange = 10.0f;
+
+	//Stats
+	uint32_t maxHealth = 3;
```

Comment:
Would recommend having health in a separate component in the future. It's likely that HP will not be unique to entities that are AI controlled.